tests/grammar/vyper.lark

```diff
@@ -196,64 +196,71 @@ dict: "{" "}" | "{" (NAME ":" _expr) ("," (NAME ":" _expr))* [","] "}"
 // Operators
 // NOTE: The recursive cycle here helps enforce operator precedence
 //       Precedence goes up the lower down you go
-?operation: bin_op
-
-_POW: "**"
-_SHL: "<<"
-_SHR: ">>"
-_BITAND: "&"
-_BITOR: "|"
-_BITXOR: "^"
-// Binary Operations
-// NOTE: Keep these in sync with aug_assign above
-?bin_op: product
-       | bin_op "+"  product -> add
-       | bin_op "-"  product -> sub
-// TODO fix precedence to match python operator precedence
-       | bin_op _SHL product -> shl
-       | bin_op _SHR product -> shr
-       | bin_op _BITAND product -> bitand
-       | bin_op _BITOR product -> bitor
-       | bin_op _BITXOR product -> bitxor
-?product: power
-        | product "*"   power -> mul
-        | product "/"   power -> div
-?power: bool_op
-      | power _POW  bool_op -> pow
-      | power "%"   bool_op -> mod
+?operation: bool_or
```

Comment:
add note that this needs to match python operator precedence - https://docs.python.org/3/reference/expressions.html#operator-precedence